src/main/java/com/google/sps/data/UserService.java

```diff
@@ -1,14 +1,8 @@
 package com.google.sps.data;
 
 import com.google.maps.GeocodingApi;
-import com.google.maps.model.AddressComponentType;
-import com.google.maps.model.AddressType;
-import com.google.maps.model.ComponentFilter;
 import com.google.maps.model.GeocodingResult;
 import com.google.maps.GeoApiContext;
-import com.google.maps.model.LatLng;
```

Comment:
Thank you for deleting all of the extra imports! I'm pretty sure that this import is needed for this call to work (LatLng)

Reply:
If you are not using the LatLng within the file, you do not need the import and I tested the code out and it was working fine after I deleted all the imports.

Reply:
Hmm interesting you're right. The object is referenced as part of the result but I guess since we're not creating a LatLng object you don't the import. Disregard!